applications/index.adoc

```diff
@@ -0,0 +1,48 @@
+
+[id="working-on-application"]
+== Working on an application
+
+[id="creating-application"]
+=== Creating an application
+To create applications, you must have created a project or have access to a project with the appropriate roles and permissions. You can create an application by using either xref:creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-developer-perspective[the Developer perspective in the web console], xref:creating_applications/creating-apps-from-installed-operators.adoc#creating-apps-from-installed-operators[installed Operators], or xref:creating_applications/creating-applications-using-cli.adoc#creating-applications-using-cli[the OpenShift Container Platform CLI]. You can source the applications to be added to the project from Git, JAR files, devfiles, or the developer catalog.
+
+You can also use components that include source or binary code, images, and templates to create an application by using the OpenShift Container Platform CLI. With the OpenShift Container Platform web console, you can create an application from an Operator installed by a cluster administrator.
+
+[id="maintaining-application"]
+=== Maintaining an application
+After you create the application you can use the web console to xref:odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc#odc-monitoring-project-and-application-metrics-using-developer-perspective[monitor your project or application metrics]. You can also xref:odc-editing-applications.adoc#odc-editing-applications[edit] or xref:odc-deleting-applications.adoc#odc-deleting-applications[delete] the application using the web console. 
+When the application is running, not all applications resources are used. As a cluster administrator, you can choose to xref:idling-applications.adoc#idling-applications[idle these scalable resources] to reduce resource consumption. 
+
+[id="connecting-application"]
+=== Connecting an application to services
+An application uses backing services to build and connect workloads, which vary as per the service provider. Using the xref:connecting_applications_to_services/understanding-service-binding-operator.adoc#understanding-service-binding-operator[Service Binding Operator], as a developer, you can bind workloads together with Operator-managed backing services, without any manual procedures to configure the binding connection. You can apply service binding also on xref:connecting_applications_to_services/getting-started-with-service-binding-ibm-power-ibm-z.adoc#getting-started-with-service-binding-ibm-power-ibm-z[IBM Power Systems, IBM Z, and LinuxONE environments].
```

Comment:
I would suggest using another phrase instead of "as per". The ISG states:

Do not use. Use a phrase such as "according to", "as", or "as in".